pkg/traces/traces.go

```diff
@@ -0,0 +1,99 @@
+package traces
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"reflect"
+	"runtime"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	semconv "go.opentelemetry.io/otel/semconv/v1.17.0"
+	"go.opentelemetry.io/otel/trace"
+)
+
+const (
+	ApplicationName           = "launcher"
```

Comment:
I wonder if we'll want to split launcher and desktop apart here. But maybe TBD?

Reply:
I'm actually not sure this is right altogether -- we use this for the name of the tracer, and the TracerProvider interface has this documentation:

```
// The passed name needs to uniquely identify instrumentation code.
// Therefore, it is recommended that name is the Go package name of the
// library providing instrumentation (note: not the code being
// instrumented).
```

So I think I should update to either `go.opentelemetry.io/otel/trace` or `github.com/kolide/launcher/pkg/traces`, and rename this const to something more appropriate.

To your point, yes we'll definitely want a way to distinguish between launcher proper and launcher desktop traces -- I haven't thought that through much yet.

Reply:
Yeah, I am going to update this -- this is used to set the `otel.library.name` tag.